README.md

```diff
@@ -1,8 +1,27 @@
+# PAGE MOVED!
+<br>
+<br>
+
+> **[!NOTE]
```

Comment:
You need to remove the ** for this to render as a note, see:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts